Sources/PointFree/Account/Cancel.swift

```diff
@@ -27,7 +27,7 @@ let cancelMiddleware =
       or: redirect(to: .account, headersMiddleware: flash(.error, "Your subscription is already canceled!"))
     )
     <| cancel
-    >-> redirect(to: .account, headersMiddleware: flash(.notice, "You’ve canceled your subscription."))
+    >-> redirect(to: .account, headersMiddleware: flash(.notice, "We’ve canceled your subscription."))
```

Comment:
I dunno how we actually wanna message this stuff, lol, but it's kinda fun to use "we".

Reply:
yeah I like this better!